.docker/release/files/etc/crontabs/root

```diff
@@ -0,0 +1,2 @@
+# Run logrotate hourly
+0 * * * * logrotate --state /tmp/logrotate.state /etc/logrotate.d/snappymail
```

Comment:
why logrotate is needed inside the container, shouldn't all logs go to STDOUT?

Reply:
I agree. In the default `php` image, `php-fpm` forces all logs to stdout, which is what we expect in a normal docker image:

```sh
$ docker run --rm -it php:8.1-fpm-alpine cat /usr/local/etc/php-fpm.d/docker.conf
[global]
error_log = /proc/self/fd/2
...
...
```

However, in the original implementation of the docker image in `.docker/release/*`, a custom `php-fpm.conf` was used that didn't include `error_log` directive, which I believe the author did so because `snappymail` generates two logs - `auth.log` which was meant to be parsed by `fail2ban` (which is not included in the image, somehow), and `error.log` (the mail `snappymail` log file). Then it tails the two of them are `tail`ed to stdout. Then `logrotate` simply deletes them whenever they reach 10MB.

In this PR, I tried to preserve the original implementation as far as possible, so I didn't remove `logrotate`.

However, when I think about `logrotate`, I don't actually see anything useful about it:
1. `fail2ban` is not running inside the container. Hence, the `auth.log` is not going to be used anyway.
2. The only log left is `error.log`, the main log file. Since there's only one log, everything might as weel go to stdout.
3. The logrotate config uses `rotate 0` which simply deletes, and does not rotate the `.log` file. 

Hence, all `logrotate` is really doing now is to ensure `snappymail` logs do not grow past 10MB.

I really feel `logrotate` should be gotten rid of and we simply log to `stdout`. What do you think?

Reply:
I think containers should follow 12factor apps patterns. They are pretty clear regarding [logs](https://12factor.net/logs). If someone will need to use fail2ban it should be a separate container.

Reply:
Either line https://github.com/the-djmaze/snappymail/blob/11a70c7e030fb7dbcebcffcf3d33988e22078966/.docker/release/files/entrypoint.sh#L60 and https://github.com/the-djmaze/snappymail/blob/11a70c7e030fb7dbcebcffcf3d33988e22078966/.docker/release/files/entrypoint.sh#L61 should be changed with something like:
```
ln -sf /dev/stderr /var/lib/snappymail/_data_/_default_/logs/errors.log
ln -sf /dev/stdout /var/lib/snappymail/_data_/_default_/logs/auth.log
```
or better to change corresponding config at https://github.com/the-djmaze/snappymail/blob/11a70c7e030fb7dbcebcffcf3d33988e22078966/.docker/release/files/entrypoint.sh#L44 and https://github.com/the-djmaze/snappymail/blob/11a70c7e030fb7dbcebcffcf3d33988e22078966/.docker/release/files/entrypoint.sh#L55 to point to `/dev/stderr` and `/dev/stdout` respectively. 
WDYT?

Reply:
> I think containers should follow 12factor apps patterns. They are pretty clear regarding [logs](https://12factor.net/logs). If someone will need to use fail2ban it should be a separate container.

Agree with 12 factor methodology, hope we get as close to to as we can.

So we have two things:
- `auth.log` to stdout, and `auth.log` to be preserved as a `.log` file for `fail2ban` in another container.
- `errors.log` goes directly to `stdout`. Doesn't need to be preserved as a `.log` file.

The only issue I see with using `sed` against `SNAPPYMAIL_CONFIG_FILE`, is that users can change the `filename` and `auth_logging_filename` values in the `Admin Panel > config` and break logs to 

Let me first remove `logrotate` and `tail -f`, then I'll try and find a way to solve this problem at the `php-fpm` level. I'd need to read the logging code a bit first.

Reply:
`[logs] filename` supports only a file name, not a full path, so we can't do this:

```sh
 sed 's/^filename = .*/filename = "/dev/stderr"/' -i $SNAPPYMAIL_CONFIG_FILE 
```

So we have to `ln` as you suggested:

```sh
ln -sf /dev/stderr /var/lib/snappymail/_data_/_default_/logs/errors.log
```

Reply:
Can't use `ln -sf`, see https://github.com/the-djmaze/snappymail/pull/965#discussion_r1111176895

Reply:
considering resolved